src/ledger/LedgerTxn.cpp

```diff
@@ -1850,6 +1850,34 @@ LedgerTxn::Impl::getWorstBestOfferIterator()
     return WorstBestOfferIterator(std::move(iterImpl));
 }
 
+bool
+LedgerTxn::HasSponsorshipEntry() const
```

Comment:
Nitpick: naming convention. Should be "hasSponsorshipEntry".

Reply:
Fixed